tests/Type/ValidationTest.php

```diff
@@ -1308,20 +1314,6 @@ private function schemaWithObjectFieldOfType($fieldType) : Schema
         ]);
     }
 
-    /**
-     * @see it('rejects an empty Object field type')
-     */
-    public function testRejectsAnEmptyObjectFieldType() : void
```

Comment:
No longer needed! PHP itself will throw an error if type is not provided and not `Type`.